node-installer/internal/config/config.go

```diff
@@ -17,6 +17,8 @@ type Config struct {
 	Files []File `json:"files"`
 	// RuntimeHandlerName is the name of the runtime handler (containerd runtime) to create.
 	RuntimeHandlerName string `json:"runtimeHandlerName"`
+	// QemuDmVerityArgs is an additional set of kernel parameters to pass kernels started with QEMU.
+	QemuDmVerityArgs string `json:"QemuDmVerityArgs"`
```

Comment:
```suggestion
	// // QemuExtraKernelParams is an additional set of kernel parameters to pass kernels started with QEMU.
	QemuExtraKernelParams string `json:"QemuExtraKernelParams"`
```

Reply:
Not a fan of specifying the whole command line. We should rather provide extra settings, I guess? Or make the settings a map.

Reply:
We need the command line the **exactly** match the one we're using to calculate the launch measurement, so my idea was to construct it ones and then use that in all required places. That way they can't go out of sync.

Reply:
Fine. But let's call it QemuKernelPrams then. It's got nothing to do with dm-verity per se